### PR TITLE
automation: build daily and report issues

### PR DESCRIPTION
**Pull request by sandrobonazzola**
## Changes introduced with this PR

* build oVirt Node daily and report build issues automatically.

## Are you the owner of the code you are sending in, or do you have permission of the owner?

[y/n] yes

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -51,53 +55,100 @@ jobs:
...
+    runs-on: ubuntu-latest
+    needs:
+      - build-matrix
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Add a comment about successful job and close issue
+        run: |
+          set -e
+          LABEL="build-failed"
```
Comment (janosdebugs) [inline .github/workflows/build.yml:120]: This label doesn't currently exists and needs to be created.

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -51,53 +55,100 @@ jobs:
...
-      run: |
-        umount -f /host/modules
-        [[ -z "${{ github.event.inputs.keep_workdir }}" ]] && rm -rf $PWD/*
+      - name: Clean up mount and entire workdir
+        if: always()
+        run: |
+          umount -f /host/modules
+          [[ -z "${{ github.event.inputs.keep_workdir }}" ]] && rm -rf $PWD/*
+
+  close-build-issue-on-success:
```
Comment (janosdebugs) [inline .github/workflows/build.yml:109]: Shouldn't this only run on scheduled builds?

**Review by janosdebugs — APPROVED** [review]
Looks good, but a few comments.

File: .github/workflows/build.yml (hunk 1 above)
Comment (sandrobonazzola) [reply .github/workflows/build.yml:120]: Done

File: .github/workflows/build.yml (hunk 2 above)
Comment (sandrobonazzola) [reply .github/workflows/build.yml:109]: would make sense, not sure how to configure that

Comment (sandrobonazzola) [reply .github/workflows/build.yml:109]: not a big deal if runs all the times and not only on the scheduled builds.

**Review by lveyde — APPROVED** [review]

**Pull request merged**